tools/src/misc/h5repart.c

```diff
@@ -227,6 +227,10 @@ main(int argc, char *argv[])
     if (argno >= argc)
         usage(prog_name);
     src_gen_name = argv[argno++];
+    if (!src_gen_name) {
+        fprintf(stderr, "invalid source file name pointer");
+        exit(EXIT_FAILURE);
+    }
```

Comment:
Even though we are just exiting anyway, valgrind is going to complain we leak the name memory (allocated above) if we ever thoroughly test the command-ilne options options.